detox/src/devices/DeviceRegistry.js

```diff
@@ -79,6 +89,12 @@ class DeviceRegistry {
       lockfilePath: environment.getDeviceLockFilePathAndroid(),
     });
   }
+
+  static forGenyCloudCleanup() {
```

Comment:
Wait, can't we stop multiplying the entities, and think, maybe, about a unified device registry?

Reply:
I think we should definitely unify the platform based separation (even added a TODO there 😄 )
But as for cleanup, for now we should keep it separated...